deployment/cfn/worker.py

```diff
@@ -420,6 +417,8 @@ def get_cloud_config(self):
                 '    content: ', self.get_input('RollbarServerSideAccessToken'),  # NOQA
                 '\n',
                 'runcmd:\n',
+                '  - mount -o "defaults,nofail,discard" -t ext4 /dev/xvdf /opt/rwd-data\n',
```

Comment:
I believe that the `cloud-config` `mount` directive also took care of persisting the mount in `/etc/fstab`. I checked out the instances on staging and did not see any entires in `fstab`, so I think we may need to replicate that functionality here.

Reply:
In 7e44ff1, I modified cloud-config to create `fstab.rwd-data`, which contains an `fstab` entry for `/dev/xvdf`.  Then I added a `runcmd` entry to add the contents of `fstab.rwd-data` to `/etc/fstab`. I figured doing it that way would make it easier to make changes and add new volumes in the future.